ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_isis.py

```diff
@@ -74,12 +74,24 @@ def router_isis(self) -> dict | None:
     @cached_property
     def _isis_net(self) -> str | None:
         isis_system_id_prefix = get(self.shared_utils.switch_data_combined, "isis_system_id_prefix")
+
         if isis_system_id_prefix is not None:
```

Comment:
Let's move everything related to the `node_id` case under that `if`. If prefix is not set, just `return None` inside that condition.